
# Analysis of the brown hagfish *Eptatretus atami*

This repository contains the code employed to investigate the hagfish genome, as well as key data files. 

## Supplementary data files

* **Supplementary File 1.** Table gathering essential  information for each gene model in E. atami including location, protein domains, gene family, gene expression cluster [Eptata_genes_filt.xlsx](https://github.com/fmarletaz/hagfish/blob/main/Functional/Eptata_genes_filt.xlsx)
* **Supplementary File 2.** Number of orthologues (mutual-best-hits) shared between hagfish and lamprey chromosomes. [Pata-Pmar_syntCounts.txt](https://github.com/fmarletaz/hagfish/blob/main/Synteny/Pata-Pmar_syntCounts.txt)
* **Supplementary File 3.** Phylogenetic trees inferred for paralogons in each CLG assuming the C20+R model   [paralogons_t6_c20.pdf](https://github.com/fmarletaz/hagfish/blob/main/Paralogons/paralogons_t6_c20.pdf)
* **Supplementary File 4.** Synteny-based paralogue classification for reconstructed gene families [Vert_Evt_OGrrA.txt](https://github.com/fmarletaz/hagfish/blob/main/Paralogons/Vert_Evt_OGrrA.txt). 

* **Supplementary File 5**  Functional enrichment for sets of paralogues showing distinct retention patterns after genome duplications in vertebrates (Fig. 4b) [Vert2R_Go_enrich_wg_gds.txt](https://github.com/fmarletaz/hagfish/blob/main/Functional/Vert2R_Go_enrich_long_wg_gds.txt). 

* **Supplementary File 6.** List of gene families labelled with their origin and lineage-specific losses [all_families_gain_loss.txt](https://github.com/fmarletaz/hagfish/blob/main/Gene%20families/all_families_gain_loss.txt)
 

Detail about analyses conducted in the paper are specified in the different subfolders. 

